chingu-voyages/v51-tier3-team-33
# Olga/feature/random UI fixes

# Description
Final UI updates  and some minor fixes 

## Main changes explained:
- Removed dark mode
- Added banner on dashboard
- Added css rule to input type number to not show up and down arrows
- Displayed message 'Add friends/groups to start sharing expenses' on the sidenav when user has no groups or friends
- Updated metadata in root layout file
- Added user expenses  sorted from most recent to oldest to recentExpenses component on dashboard
- Added footer with group info and link to github
- Removed date input from NewExpense form


## How to test:
1. check into current branch
2. do `pnpm install` and `pnpm run dev` to run this PR locally
3. log in to app
4. check that everything works normally and looks nice
5. set dark mode on your device and verify that the app does not switch to dark mode
